cli/cmd/uninject.go

```diff
@@ -65,7 +65,7 @@ sub-folders, or coming from stdin.`,
 func (rt resourceTransformerUninject) transform(bytes []byte) ([]byte, []inject.Report, error) {
 	conf := inject.NewResourceConfig(rt.configs)
 
-	report, err := conf.ParseMetaAndYaml(bytes)
+	report, err := conf.ParseMetaAndYAML(bytes)
```

Comment:
Use uppercase for acronym. Refer [golang wiki](https://github.com/golang/go/wiki/CodeReviewComments#initialisms).